Lib/List.hpp

```diff
@@ -649,6 +649,7 @@ class List
 	return;
       }
 
+      ASS_NEQ(_prev,0);
```

Comment:
In this case we know that `_prev` must not be `nullptr`, but it's not a trivial invariant in some cases.